change/@fluentui-react-charting-743e453a-27eb-4dbc-9715-a1eeeefa66e5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
pls remove this changefile as its not needed anymore

Reply:
it will be needed to re-publish without the transform.

Reply:
done :)